hack/lib/golang.sh

```diff
@@ -480,8 +480,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(GOFLAGS='' go version)"
   local minimum_go_version
-  # TODO(liggitt): Need to switch this to 1.19 once we update images to newer go version
-  minimum_go_version=go1.17.0
```

Comment:
in order to support running unit/integration tests on the original go version for 1.23 (xref https://github.com/kubernetes/test-infra/pull/28626), I think we should perhaps leave this at 1.17